packages/flutter/lib/src/painting/inline_span.dart

```diff
@@ -407,3 +434,287 @@ abstract class InlineSpan extends DiagnosticableTree {
     style?.debugFillProperties(properties);
   }
 }
+
+/// A text attribute set that can be used to update the attributes of an
+/// [InlineSpan] subclass.
+///
+/// All attributes are nullable, and default to `null`. Attributes set to `null`
+/// in an [InlineSpanAttributes] will not be updated.
+///
+/// See also:
+///
+///  * [InlineSpan.updateAttributesAtOffset], which takes an [InlineSpanAttributes], and
```

Comment:
Here and elsewhere in the docs: Should the docs link to `updateAttributes` instead?